chapter-08/end/Program.cs

```diff
@@ -19,9 +18,8 @@ public static class Program
         [STAThread]
         static void Main()
         {
-            using (var game = new MainGame(WIDTH, HEIGHT, new GameplayState()))
+            using (var game = new MainGame(WIDTH, HEIGHT, new SplashState()))
```

Comment:
Intentional to swap the starting state to Splash instead of Gameplay?

Reply:
Yes, at some point while I was writing I noticed that I stopped using the splash screen, but never went back to check previous chapters. I think that started right around when I started using the DevState class.